qsignature/src/org/qcmg/sig/model/Comparison.java

```diff
@@ -107,15 +84,18 @@ public String toString() {
 		sb.append(test).append(" (").append(testCoverage).append(") : ");
 		sb.append(Double.isNaN(score) ? "NaN" : score);
 		sb.append(", ").append(overlapCoverage);
-//		sb.append(", ").append(numberOfCalculations);
 		return sb.toString();
 	}
 
 	@Override
 	public int compareTo(Comparison o) {
 		
-		if (score < o.score) return -1;
-		if (score > o.score) return 1;
+		if (score < o.score) {
```

Comment:
suggest to change:
if (score == o.score) {
    return overlapCoverage - o.overlapCoverage;
}
return (score> o.score )? 1 : -1;

Reply:
done